.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Ok, commit name better but please put at beginning ticket number/ code like here: TASK-LINKS-1 And then your comments